src/translations/fr.json

```diff
@@ -156,6 +156,21 @@
     "FiltersListsSelection": "Sélection des listes de filtres",
     "sensitivityAnalysisResults": "Résultats de l'analyse de sensibilité",
     "sensitivityAnalysisError": "Une erreur est survenue lors de l'exécution de l'analyse de sensibilité",
+    "SensitivityInDeltaMW":"Sensibilité au niveau des quadripôles en ΔMW",
+    "SensitivityInDeltaA":"Sensibilité au niveau des quadripôles en ΔA",
+    "SensitivityAtNode" : "Sensibilité au niveau des noeuds",
+    "ValRefInkWBeforeContingency" : "Puissance active de référence (MW) avant aléa",
```

Comment:
You could have a translation key as : 
```
"BeforeContingency" : "before contingency",
```
and append it to `ValRefInkW` , `DeltaInkW`, ... etc when needed